### PR TITLE
epinio 1.2.1 - get the bash working for our staging

### PR DESCRIPTION
**Pull request by andreas-kupries**
#### Pull Request Checklist ####

- [x] Change does not remove any existing Images or Tags in the images-list file
- [x] Change does not remove / overwrite exiting Images or Tags in Rancher DockerHub
- [ ] If updating an existing entry, verify the `SOURCE` is still accurate and upstream hasn't been migrated to a new regitry or repo (if they've migrated, a new repo request to EIO is needed to comply with the `SOURCE DESTINATION TAG` pattern)
- [x] New entries are in format `SOURCE DESTINATION TAG`
- [x] New entries are added to the correct section of the list (sorted lexicographically)
- [ ] New entries have a repo created in Rancher Dockerhub (where the image will be mirrored to)
- [ ] Changes to scripting or CI config have been tested to the best of your ability

#### Types of Change ####

New images (tags of existing bases):

  - `rancher/mirrored-epinio-epinio-unpacker 1.0`

    Note: https://github.com/rancherlabs/eio/issues/1309 is the EIO PR asking for the creation of a `rancher/mirrored-epinio-epinio-unpacker` repository

#### Linked Issues ####

Ref: https://github.com/rancher/charts/pull/1814
Ref: https://github.com/epinio/epinio/issues/1248#issuecomment-1213074682

#### Final Checks after the PR is merged ####
- [ ] Confirm that you can pull the new images and tags from DockerHub

### CODE REVIEW

**Review by brandond — DISMISSED** [review]

File: images-list [hunk 1]
```diff
@@ -369,6 +369,7 @@ k8s.gcr.io/sig-storage/livenessprobe rancher/mirrored-sig-storage-livenessprobe
 kiwigrid/k8s-sidecar rancher/kiwigrid-k8s-sidecar 0.1.151
 kiwigrid/k8s-sidecar rancher/kiwigrid-k8s-sidecar 1.1.0
 kiwigrid/k8s-sidecar rancher/mirrored-kiwigrid-k8s-sidecar 0.1.151
+library/bash rancher/mirrored-library-bash 5.1.4
```
Comment (cbron) [inline images-list:372]: This image, and bash in general, is [licensed as GPL3](https://hub.docker.com/_/bash). Before merging we would need to confirm that this specific image is fine to incorporate into the product.

**Review by cbron — CHANGES_REQUESTED** [review]

File: images-list (hunk 1 above)
Comment (macedogm) [reply images-list:372]: We are checking internally about this (at the moment I'm waiting to confirm some meetings).

If only bash is needed, I would recommend to use [bci/bci-minimal](https://registry.suse.com/static/bci/bci-minimal-15sp4/index.html) instead.

Comment (andreas-kupries) [reply images-list:372]: Discussion running slack.

Comment (andreas-kupries) [reply images-list:372]: Currently trialing a replacement which should not have legal issues. See https://github.com/epinio/epinio/pull/1741 and https://github.com/epinio/helm-charts/pull/276

Comment (andreas-kupries) [reply images-list:372]: Ah, we crossed here @macedogm  GH did not show me your comment when you made it. Might have shaved a bit of time ... @pgonin also brought BCI up and from there the research/work got started.

Comment (andreas-kupries) [reply images-list:372]: Current plan is to have the image as `ghcr.io/epinio/epinio-unpacker:TAG-TO-BE-DETERMINED`

Comment (andreas-kupries) [reply images-list]: Tag is 1.0. Force pushed new commit using `ghcr.io/epinio/epinio-unpacker:1.0`.

THE PR https://github.com/rancherlabs/eio/issues/1309 is still the necessary EIO request.

**Review by brandond — APPROVED** [review]

**Review by cbron — APPROVED** [review]
lgtm, will merge on monday

**Review by macedogm — APPROVED** [review]

**Pull request merged**